lua/textcase/shared/utils.lua

```diff
@@ -104,10 +104,11 @@ local callableTable = {
   end
 }
 
-function utils.create_wrapped_method(desc, method)
+function utils.create_wrapped_method(method_name, method, desc)
```

Comment:
This is a breaking change, but the utils are kind of internal I guess.

A search in Github reveals no other hits than this repo or forks of it: https://github.com/search?q=create_wrapped_method+language%3ALua&type=code&l=Lua

But this won't reveal usages in private repos of course.